### DISCUSSION TITLE
关于Timer的问题

### DISCUSSION THREAD

**Discussion by Jilingwei** [cloudwu/skynet#1521] [General]
云大你好，我看到在skynet_timer中timer_update里面的timer_execute执行了两次，我在之前的Issues中看到您的回复说
“timeout 0 的消息怎么处理，并没有对 timer 这个模块做出什么承诺。”
我是不是可以理解为，防止用户直接使用 timer_add 方法主动添加 timer0的情况。
望云大能抽空回复，万分感谢！
最后祝云大身体健康，万事如意。

**Comment by cloudwu** [comment]
是说，这个模块只负责自己的事情，不管其它模块是怎么设计和使用的。

**Reply by Jilingwei** [reply]
好的，感谢云大！
